ballerina/utils.bal

```diff
@@ -188,6 +188,20 @@ isolated function generateCanonicalHeaders(map<string> headers, http:Request? re
     return [canonicalHeaders, signedHeaders];
 }
 
+# Function to generate signing key.
+#
+# + secretAccessKey - Value of the secret key.
+# + shortDateStr - shortDateStr Parameter Description
+# + region - Endpoint region.
+# + return - Signing key.
```

Comment:
```suggestion
# + secretAccessKey - Value of the secret key
# + shortDateStr - shortDateStr Parameter Description
# + region - Endpoint region
# + return - Signing key
```
To adhere with [Ballerina documentation style](https://ballerina.io/learn/generate-code-documentation/#write-ballerina-documentation)